tensorflow/core/kernels/relu_op.cc

```diff
@@ -143,11 +143,13 @@ namespace functor {
       typename TTypes<T>::Tensor backprops);                                   \
   extern template struct SeluGrad<GPUDevice, T>;
 
+#if GOOGLE_CUDA
```

Comment:
Could you add a comment that this is currently not supported on ROCm?

Reply:
done